src/identity/json.rs

```diff
@@ -0,0 +1,57 @@
+use crate::identity::AccessResponse;
+use crate::identity::IdentityStore;
+use crate::identity::Outcome;
+use crate::util::Result;
+use std::path::Path;
+use std::fs::File;
+
+#[derive(Deserialize)]
+pub struct JsonSettings {
+    pub file_path: String,
+}
+
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Why?

Reply:
Renamed the variables during development. Now it is not needed anymore. Will remove.